src/test/java/no/nav/fo/veilarbregistrering/arbeidsforhold/ArbeidsforholdTest.java

```diff
@@ -39,7 +39,11 @@ public void datoSkalIkkeVaereInneforPeriode() {
         LocalDate mnd = LocalDate.of(2017,12,1);
         LocalDate fom = LocalDate.of(2017,9,1);
         LocalDate tom = LocalDate.of(2017,11,30);
-        Arbeidsforhold arbeidsforhold = new Arbeidsforhold().setFom(fom).setTom(tom);
+        Arbeidsforhold arbeidsforhold = arbeidsforholdMedDato(fom, tom);
         assertThat(arbeidsforhold.erDatoInnenforPeriode(mnd)).isFalse();
     }
+
+    private Arbeidsforhold arbeidsforholdMedDato(LocalDate fom, LocalDate tom) {
```

Comment:
Ikke så mye å hente, men har en ArbeidsforholdTestdataBuilder for noen "object mothers" for arbeidsforhold.

Jeg tenker det kan være fint å bruke, når vi strammer inn hva et gyldig arbeidsforhold ser ut. At f.eks. org.nr kanskje må være fylt ut. Det gir oss også mulighet til å bruke metoder på objektet som er begrenset til package i test, ved at vi går via ArbeidsforholdTestdataBuilder som ligger i samme pakke som Arbeidsforhold.

Reply:
God idé – takk! 😃